### PR TITLE
operators syndesis (1.13.0)

### PR DESCRIPTION
**Pull request by zregvart**
### Updates to existing Operators

* [x] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [x] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [ ] Have you tested an update to your Operator when deployed via OLM?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [ ] Modify more than one operator
* [ ] Modify an Operator you don't own
* [ ] Rename an operator - please remove and add with a different name instead
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [x] Description about the managed Application and where to find more information
2. [x] Features and capabilities of your Operator and how to use it
3. [x] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [x] Human readable name and 1-liner description about your Operator
* [x] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [x] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [x] Links to the maintainer, source code and documentation
* [x] Example templates for all Custom Resource Definitions intended to be used
* [x] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Comment by J0zi** [comment]
@zregvart chek this - Error: [syndesis/1.13] Value ''metadata.annotations.alm-examples'' in CSV is not valid json object. Currently found on kiwi test https://github.com/redhat-openshift-ecosystem/community-operators-prod/runs/4596260029?check_suite_focus=true#step:6:618

**Comment by zregvart** [comment]
@J0zi thanks, after a couple days of debugging this locally I did find the root cause of the issue it was a space instead of `T` in `createdAt`:

```diff
diff --git a/operators/syndesis/1.13.0/manifests/syndesis.clusterserviceversion.yaml b/operators/syndesis/1.13.0/manifests/syndesis.clusterserviceversion.yaml
index bf376c84..6667fbcb 100644
--- a/operators/syndesis/1.13.0/manifests/syndesis.clusterserviceversion.yaml
+++ b/operators/syndesis/1.13.0/manifests/syndesis.clusterserviceversion.yaml
@@ -7,7 +7,7 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Integration & Delivery
     certified: "false"
-    createdAt: '2021-12-18 12:00:00Z'
+    createdAt: '2021-12-18T12:00:00Z'
     containerImage: quay.io/syndesis/syndesis-operator:1.13
     support: Syndesis
     description: Manages the installation of Syndesis, a flexible and customizable
```

Which threw off the `from_yaml` so that the metadata wasn't in expected syntax. That is it was this:

```
  ansible_facts:
    operator_vars:
      metadata:
        annotations: '{''capabilities'': ''Seamless Upgrades'', ''categories'': ''Integration & Delivery'', ''certified'': ''false'', ''createdAt'': datetime.datetime(2021, 12, 18, 12, 0, tzinfo=datetime.timezone.utc), ''containerImage'': ''quay.io/syndesis/syndesis-operator:1.13'', ''support'': ''Syndesis'', ''description'': ''Manages the installation of Syndesis, a flexible and customizable open source platform that provides core integration capabilities as a service.'', ''repository'': ''https://github.com/syndesisio/syndesis/'', ''alm-examples'': ''[{\n  "apiVersion": "syndesis.io/v1beta3",\n  "kind": "Syndesis",\n  "metadata": {\n    "name": "app"\n  },\n  "spec" : {\n    "addons": {\n      "jaeger": {\n        "enabled": true\n      },\n      "knative": {\n        "enabled": false\n      },\n      "ops": {\n        "enabled": false\n      },\n      "publicApi": {\n        "enabled": false\n      },\n      "todo": {\n        "enabled": false\n      }\n    }\n  }\n}]''}'
```

Instead of this:

```
  ansible_facts:
    operator_vars:
      metadata:
        annotations:
          alm-examples: |-
            [{
              "apiVersion": "syndesis.io/v1beta3",
              "kind": "Syndesis",
              "metadata": {
                "name": "app"
              },
              "spec" : {
                "addons": {
                  "jaeger": {
                    "enabled": true
                  },
                  "knative": {
                    "enabled": false
                  },
                  "ops": {
                    "enabled": false
                  },
                  "publicApi": {
                    "enabled": false
                  },
                  "todo": {
                    "enabled": false
                  }
                }
              }
            }]
```

So something to recall if you see a similar issue in the future.

Also I've found that the kind version installed (0.9.0) doesn't work on Fedora 35 or on Ubuntu 22.04 and upgrading to 0.11.1 helped with that.

The several commits today were me trying to use the version in the form `MAJOR.MINOR` or `MAJOR.MINOR.x` and test suite not having it, so I settled on `MAJOR.MINOR.0` which seemed to have worked.

If there is any way for us not having to create a new bundle/manifests/version for a patch releases I'd like to do that rather than to create new PRs for every patch release we do. If you can point me to a way of doing that it would be very helpful.

File: operators/syndesis/1.13.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=syndesis
+LABEL operators.operatorframework.io.bundle.channels.v1=1.13.0
```
Comment (phantomjinx) [inline operators/syndesis/1.13.0/Dockerfile:8]: Should the channels not be 1.13.x
See -> https://github.com/redhat-openshift-ecosystem/community-operators-prod/blob/main/operators/syndesis/1.12.0/Dockerfile#L8

File: operators/syndesis/1.13.0/metadata/annotations.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: syndesis
+  operators.operatorframework.io.bundle.channels.v1: '1.13.0'
```
Comment (phantomjinx) [inline operators/syndesis/1.13.0/metadata/annotations.yaml:7]: Same as above.

**Review by phantomjinx — CHANGES_REQUESTED** [review]

**Review by phantomjinx — APPROVED** [review]

**Pull request merged**